### PR TITLE
[Helm Chart] allow bucket metrics retrieve

### PR DESCRIPTION
**Pull request by mmadoo**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license] (https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
Buckets metrics are now on endpoint `/minio/v2/metrics/bucket ` and no more `/minio/v2/metrics/cluster`.
This PR allows to configure a prometheus in order to have this endpoint scraped.

## How to test this PR?
This PR can be test by deploying helm chart on a kubernetes cluster with an prometheus operator with
```
helm upgrade --install  --set resources.requests.memory=512Mi --set replicas=1 --set persistence.enabled=false --set mode=standalone --set metrics.serviceMonitor.enabled=true --set metrics.serviceMonitor.includeBucket=true --set metrics.serviceMonitor.interval=1m --set rootUser=rootuser,rootPassword=rootpass123  
release .
```
It should create a new probe like:
```
apiVersion: monitoring.coreos.com/v1
kind: Probe
metadata:
  labels:
    app: minio
    app.kubernetes.io/managed-by: Helm
    chart: minio-5.0.14
    heritage: Helm
    release: release
spec:
  interval: 1m
  jobName: release-minio
  prober:
    path: /minio/v2/metrics/bucket
    scheme: http
    url: release-minio.nicolas:9000
  targets:
    staticConfig:
      static:
        - release-minio.nicolas
```

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

**Review by shtripat — COMMENTED** [review]
There is another endpoint `/minio/v2/metrics/node` for node metrics. Kindly add that as well.
Also in steps for testing mention that it needs prometheus operator to be deployed in your k8s cluster.

**Comment by mmadoo** [comment]
> There is another endpoint `/minio/v2/metrics/node` for node metrics. Kindly add that as well. Also in steps for testing mention that it needs prometheus operator to be deployed in your k8s cluster.

Scapping of this endpoint was already possible by setting the property `metrics.serviceMonitor.includeNode` to `true`

**Comment by shtripat** [comment]
> > There is another endpoint `/minio/v2/metrics/node` for node metrics. Kindly add that as well. Also in steps for testing mention that it needs prometheus operator to be deployed in your k8s cluster.
> 
> Scapping of this endpoint was already possible by setting the property `metrics.serviceMonitor.includeNode` to `true`

Oh, I see that now. Thanks!

**Review by shtripat — APPROVED** [review]
lgtm

File: helm/minio/values.yaml [hunk 1]
```diff
@@ -530,6 +532,8 @@ metrics:
       # metricRelabelings:
       #   - regex: (server|pod)
       #     action: labeldrop
+    # for bucket metrics
+    relabelConfigsBucket: {}
```
Comment (harshavardhana) [inline helm/minio/values.yaml:536]: Where is this used ?

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Pull request closed without merging**